sdk/search/azure-search-documents/azure/search/documents/_index_documents_batch.py

```diff
@@ -3,24 +3,19 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-from typing import TYPE_CHECKING
+from typing import Union, List, Dict, Any
 from threading import Lock
 
 from ._generated.models import IndexAction
 
-if TYPE_CHECKING:
-    # pylint:disable=unused-import,ungrouped-imports
-    from typing import List
 
-
-def _flatten_args(args):
-    # type (Union[List[dict], List[List[dict]]]) -> List[dict]
+def _flatten_args(args: Union[List[Dict], List[List[Dict]]]) -> List[Dict]:
```

Comment:
Same for the dicts here. Are the keys at least well defined to be strs ?

Reply:
I am not 100% sure. There are documents and it may be possible that some properties are not strings?

Reply:
@kristapratico is there any meaningful difference between `Dict` and `Dict[Any, Any]` in this case?

Reply:
No difference for the type checkers - https://mypy-play.net/?mypy=latest&python=3.11&gist=c5651f063e241f5da69333782331fd0e